js/damage.js

```diff
@@ -69,7 +69,7 @@ function getDamageResult(attacker, defender, move, field) {
     }
     
     var defAbility = defender.ability;
-    if (["Mold Breaker", "Teravolt", "Turboblaze"].indexOf(attacker.ability) !== -1) {
+    if ((["Mold Breaker", "Teravolt", "Turboblaze"].indexOf(attacker.ability) !== -1) || move.name === "Moongeist Beam" || move.name === "Sunsteel Strike") {
```

Comment:
The body of the if statement does not only ignore the defender's ability, but also record attacker's ability in the result message, so I think it would be better to put Moongeist Beam check in another if statement